packages/flutter_tools/lib/src/debug_adapters/README.md

```diff
@@ -34,9 +34,9 @@ Arguments specific to `launchRequest` are:
 - `bool? noDebug` - whether to run in debug or noDebug mode (if not supplied, defaults to debug)
 - `String program` - the path of the Flutter application to run
 - `List<String>? args` - arguments to be passed to the Flutter program
-- `List<String>? toolArgs` - arguments for the `flutter` tool
-- `String? console` - if set to `"terminal"` or `"externalTerminal"` will be run using the `runInTerminal` reverse-request; otherwise the debug adapter spawns the Dart process
-- `bool? enableAsserts` - whether to enable asserts (if not supplied, defaults to `true`)
```

Comment:
`console` and `enableAsserts` were removed here as these had been incorrectly included because a lot of this file started as a copy/paste of Dart. We do not support running in external terminals for Flutter (since we need the JSON output from), and asserts are already controlled by Flutter depending on whether it's debug/release mode.